lending/loan_management/doctype/loan/test_loan.py

```diff
@@ -2080,6 +2080,84 @@ def test_loan_restructure_schedule_with_bpi_adjustment(self):
 
 		self.assertEqual(second_repay_schedule_current_principal_amount, second_adjustment_after_pos)
 
+	def test_dpd_calculation_for_non_loc_loan_without_disbursement(self):
```

Comment:
The test case still passes without any changes related to the fix
Doesn't seem to be doing proper testing